dipy/core/geometry.py

```diff
@@ -126,7 +126,11 @@ def cart2sphere(x, y, z):
        azimuth angle
     '''
     r = np.sqrt(x * x + y * y + z * z)
-    theta = np.arccos(z / r)
+    # r is strictly positive by construction and will be 0 iff all three are
+    # zero, so it's safe to convert it to a 1 in this case, and prevents
+    # warnings from being raised in the following statement (z / r):
+    r = np.where(r > 0, r, 1)
```

Comment:
This part is not anymore necessary after the last changes :-)

Reply:
Any chance to see this PR in the future release?  You just need to remove the lines from 129 to 132 and keep your change line 133.

Reply:
Oh - I didn't understand that was all that was left. Done and rebased!